packages/datadog-plugin-pg/test/index.spec.js

```diff
@@ -112,6 +112,28 @@ describe('Plugin', () => {
             })
           })
 
+          if (version === '4.0.0') {
```

Comment:
If the code applies for all versions the test should apply to all versions as well.

Reply:
well we test version 4 and version 8, and it doesn't apply to version 8, the whole error eventemitter seems to be gone by then.

Reply:
Then why are we adding the emit on version 8? In what exact version did the query stop being an emitter?